config.TEMPLATE.inc.php

```diff
@@ -50,6 +50,11 @@
 ; To set the "Secure" attribute for the cookie see the setting force_ssl at the [security] group
 session_samesite = Lax
 
+; Enable this if you want to enable cookie encryption
+; The length of cookie encryption key lenght must be 16
+; Note that updating or removing cookie encryption key will result in logout from all devices
+; session_cookie_enctyption_key = ''
```

Comment:
Typo: should be `encryption`, not `enctyption`. (Probably needs syncing elsewhere.)